extensions/data-plane/data-plane-framework/src/main/java/org/eclipse/dataspaceconnector/dataplane/framework/DataPlaneFrameworkExtension.java

```diff
@@ -50,6 +55,16 @@
     private DataPlaneManagerImpl dataPlaneManager;
     private Monitor monitor;
 
+    @Inject(required = false)
+    private TransferServiceSelectionStrategy transferServiceSelectionStrategy;
+
+    public DataPlaneFrameworkExtension() {
+    }
+
+    DataPlaneFrameworkExtension(TransferServiceSelectionStrategy transferServiceSelectionStrategy) {
+        this.transferServiceSelectionStrategy = transferServiceSelectionStrategy;
+    }
```

Comment:
These two constructors seem useless.

Reply:
One is for UT, with #843 we will be able to remove this :) That was the driver for #843

Reply:
Seems #843 has now been merged, can we remove these constructors?

Reply:
@bscholtes1A please check out updates I just did to DataPlaneFrameworkExtensionTest indeed :)